data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3433 @@
+			`	When landing on an Unfettered planet, even after having brought them a jump drive, there is always some lingering tension. Now you feel the tension much more than usual, probably because of the medical emergency. Unfettered are all around, various vehicles coming and going frequently to bring the wounded to the hospital. The hospital itself looks different than the rest of the Unfettered structures, it seems more organized, proper. It is probably the best maintained structure here alongside the shipyards, or at least it has that appearance. One part of it looks to have been grafted onto the rest though, and not by a well trained physician, to say the least... Not even their critical buildings appear to be totally safe from the Unfettered's propensity for haphazard building extensions.`
+			`	Considering all of this, the Unfettered may not take no for an answer... Plenty of ships are already taking off to try and get those much needed medicals.`
+			choice
+				`	(Signal them I am going to get the supplies.)`
+					accept
+				`	(I do not have time for this.)`
+			`	Thankfully it seems that everyone is too busy with the local problem to annoy you or press you into accepting this mission, but this will probably have consequences later. You better not stay landed for too long, or you could draw unwanted attention.`
+	on visit
+		dialog phrase "generic cargo on visit"
+	on stopover
+		dialog `You are surprised to see the Hai are more than happy to give you the medical supplies needed. They load up the cargo and once they are done one of they addresses you, "Tell our brothers and sisters to stay strong, hopefully it is not too late," he says with determination.`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on complete
+		log `Brought medial supplies to Darkcloak when it was in great need of them, to prevent an epidemic.`
+		log "People" "Sha-Ri" `An Unfettered Chief worried for his people, inside Hai or Unfettered territory. He seems to believe that all Hai are equal, and as a member of the council he probably has quite some influence.`
+		payment 1500000
+		conversation
+			`The staff seem greatly relieved when you return promptly with the medical supplies. A local administrator takes the time to track you down and give you <payment>.`
+			`	As you head to leave, you notice the same Hai Chief speaking with someone who could be the hospital's chief medical officer judging by their demeanour and attire. He soon sees you as well and moves to speak with you, seeming more than a bit emotional, "I knew you could be counted on as a friend to the Unfettered! With the swift response of captains like you, it seems like it has calmed down. Maybe we worried too much? We did not before have the time to get properly introduced. I am Sha-Ri." He makes a little gesture, you assume of deference. You try to imitate the motion, apparently badly, amusing Sha-Ri. "Worry not, <first> <last>, I already know who you are."`
```

Comment:
It is the Chief from the mission before; and at the end he said you should come to Darkcloak because he thought there was a problem there
I'll just say it's Sha-Ri directly